tests/console/pciutils.pm

```diff
@@ -0,0 +1,31 @@
+#SUSE"s openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: pciutils
+# Summary: PED-8229, Introduce smoke tests for PCI Utils
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    select_serial_terminal;
+
+    add_suseconnect_product('sle-module-development-tools') if is_sle('>=15');
+    zypper_call('in cpupower powertop pciutils libpci3');
```

Comment:
IMO it does not make sense to install libraries that way. The sover will change at some point.

IMO just use `pciutils` if you need a version number.